.gitignore

```diff
@@ -34,3 +34,8 @@
 
 # Bazel files
 /bazel-*
+/.idea/
```

Comment:
This seems to be IDE specific files, would you put them in a separate section (and put a comment)?

Reply:
Yes, I'll do that right away. Actually, do you mind to see the other PR I've made sometime before?

> https://github.com/open-telemetry/opentelemetry-cpp/pull/129

It contains the header I'd like to merge into while this PR is more about consulting about some of the coding details.

Reply:
They should have been removed by now.

Reply:
Do you still plan to keep this line?

Reply:
I've commited those changes to remove it.